d2l-input-time.html

```diff
@@ -0,0 +1,392 @@
+<link rel="import" href="../polymer/polymer.html">
+<link rel="import" href="d2l-input-shared-styles.html">
+<link rel="import" href="../d2l-dropdown/d2l-dropdown.html">
+<link rel="import" href="../d2l-dropdown/d2l-dropdown-content.html">
+<link rel="import" href="../iron-a11y-keys-behavior/iron-a11y-keys-behavior.html">
+<link rel="import" href="../iron-selector/iron-selector.html">
+<link rel="import" href="../d2l-typography/d2l-typography-shared-styles.html">
+<link rel="import" href="../d2l-intl-import/d2l-intl.html">
+
+<!--
+`d2l-input-time`
+Accessible, Localized Time Picker Input Element
+
+@demo demo/d2l-input-time.html
+-->
+
+<dom-module id="d2l-input-time">
+	<template strip-whitespace>
+		<style include="d2l-input-styles">
+			:host {
+				--d2l-input-line-height: 1;
+				--d2l-input-padding: 0.7rem 1rem;
+				--d2l-input-padding-focus: calc(0.7rem - 1px) calc(1rem - 1px);
+
+				display: inline-flex;
+				flex-direction: column;
+			}
+
+			input {
+				font-family: inherit;
+			}
+
+			.d2l-dropdown-opener {
+				max-width: 6rem;
+			}
+
+			d2l-dropdown-content {
+				--d2l-dropdown-verticaloffset: 10px;
+			}
+
+			.timezone {
+				@apply --d2l-body-compact-text;
+				color: var(--d2l-color-tungsten);
+				font-style: italic;
+				padding-top: 10px;
```

Comment:
Seems like maybe we should follow rule of "6"s.  Not sure how much of that is up for grabs though.

Reply:
This is used to center the timezone text beside the `dropdown`, so I think that would require changes to `dropdown` as well?

Reply:
Ah ok.  It's a minor thing.  If one day dropdown is updated for the design spacing changes, this would then need to be updated too.  Something seemingly small that has ripple effects.  :(

Reply:
I misunderstood where the timezone was supposed to go, so this doesn't actually rely on the height of the dropdown.  So we can change the value, but then it will cause a visual difference for anyone that pulls in the component.  Are we good with that, or do we need to talk to design?